docs/website/blog/2022-12-13-odo-v3.4.0.md

```diff
@@ -50,7 +50,6 @@ Example:
 #### Using `odo` with Podman
 
 Based on your valuable feedback, we have worked on improving the experience of using `odo` with Podman.
```

Comment:
I believe it's fine to keep this as it is (by removing link to the experimental mode doc) since at the time of v3.4.0 Podman was still in the experimental mode.